code/datums/gamemodes/crash.dm

```diff
@@ -153,26 +153,21 @@
 		larva_check_interval = world.time + 1 MINUTES
 		var/datum/hive_status/normal/xeno_hive = GLOB.hive_datums[XENO_HIVE_NORMAL]
 		var/list/living_player_list = count_humans_and_xenos(count_flags = COUNT_IGNORE_HUMAN_SSD)
-		var/num_humans = living_player_list[1]
 		var/num_xenos = living_player_list[2] + xeno_hive.stored_larva
+		var/total_jobworth = get_total_joblarvaworth()
+		latejoin_larvapoints = total_jobworth / (max(1, num_xenos) * latejoin_larvapoints_required)
```

Comment:
If undefined, `latejoin_larvapoints_required` will be zero and cause a division by zero error here.

Reply:
It's defined as zero at the top of the file